QuizBowlDiscordScoreTracker/TeamManager/IByRoleTeamManager.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace QuizBowlDiscordScoreTracker.TeamManager
+{
+    public interface IByRoleTeamManager
+    {
+        public void ReloadTeamRoles(out string message);
```

Comment:
Nitpick: "public" isn't needed here, since it's an interface

Reply:
solved

Reply:
The interface itself should be public. The method doesn't need to be public.